src/com/layout/col/col.js

```diff
@@ -0,0 +1,22 @@
+import {h, Component} from 'preact/preact';
+
+export default class LayoutCol extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render( props ) {
+    let {flex = 1, flexGrow, flexShrink, flexBasis = 100} = props;
+
+    flexGrow = (flexGrow != null) ? flexGrow : flex;
+    flexShrink = (flexShrink != null) ? flexShrink : flex;
+
+    let style = "flex: " + flexGrow + " " + flexShrink + "" + flexBasis + "%; max-width: " + flexBasis + "%;";
+
```

Comment:
Does this style stuff work when you resize the window?

Reply:
It sure does, it's all based on percentages.

Reply:
Cool